code/datums/mood_events/generic_negative_events.dm

```diff
@@ -332,10 +332,10 @@
 	mood_change *= people_laughing_at_you
 	return ..()
 
-//These are unused so far but I want to remember them to use them later
 /datum/mood_event/surgery
 	description = "THEY'RE CUTTING ME OPEN!!"
 	mood_change = -8
+	timeout = 10 MINUTES
```

Comment:
time is a placeholder. my mindset is that irl waking up during surgery is something that DOES cause PTSD. feel free to debate lower times.